ciao-launcher/main.go

```diff
@@ -333,6 +331,22 @@ func connectToServer(doneCh chan struct{}, statusCh chan struct{}) {
 		dialCh <- err
 	}()
 
+	var clusterConfig payloads.Configure
+	var data []byte
+	for {
+		data = client.ClusterConfiguration()
+		if data != nil {
+			break
+		}
+	}
+	err := yaml.Unmarshal(data, &clusterConfig)
```

Comment:
I just pushed commit 7d9337799a7442190a0b635da0ea3aa16298382d that changes the client.ClusterConfiguration() call to return a payloads.Configure structure instead.